oe_content.module

```diff
@@ -137,5 +137,13 @@ function oe_content_node_prepare_form(EntityInterface $entity, $operation, FormS
 
   if (!empty($content_owners)) {
     $entity->set('oe_content_content_owner', array_values($content_owners));
+    // Set default corporate body entity for authors field.
```

Comment:
Earlier in the method we check whether the entity has the "oe_content_content_owner" field or not and we return early if it does. This means that hypothetically a content that has a "oe_author" field but not a "oe_content_content_owner" field would not get the default value.